core/monitor/verify.go

```diff
@@ -55,6 +57,15 @@ var (
 	ErrNotMatchingMapRoot = errors.New("recreated root does not match")
 )
 
+func appendErr(errList []error, s *status.Status, err error) []error {
```

Comment:
I was idly wondering whether:
 * `errList` might be a type:   `type errList []error`
 * `append` might be a method for an `errList` receiver:  func (el *errList) append(e error) {}`
So the calls in the code further down would be:
```
errs := &errList{}
errs.append(s, err)
...
```
As in this playground: https://play.golang.org/p/Lus2oqao8X

But actuallly I think it's uglier.  So nvm :)

Reply:
I actually like it.